Kconfig

```
@@ -30,6 +30,11 @@ config IMAGE_SWUPDATE
 	  Based on the example image, this adds SWUpdate and changes the
 	  partition layout to an A/B rootfs.
 
+config IMAGE_QEMU
+	bool "QEMU Image"
+	help
+	  QEMU image to boot image without real hardware.
```

Should mention the differences to the real device, e.g. the missing Arduino I/Os.

---

```suggestion
	  QEMU image to boot image without real hardware. This image has
          missing or not working features especially dependent on hardware.
          Please consider limited functionality before using this (e.g. the
          missing Arduino I/Os).
```

Is this information enough/clear? What could I add more?

---

What else is different? We only have one Ethernet port configured via the command line below. Only one UART, the debug one.

I would write it like this: "The device emulated by QEMU is not identical to a physical IOT2050. This leads to missing features such as ... Please be aware of potential problems when using the image for testing purposes."

---

Done.